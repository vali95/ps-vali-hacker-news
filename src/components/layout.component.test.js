import React from 'react'
import {shallow} from 'enzyme'

import Layout from './layout.component'

describe("Layout component test",()=>{
    const props = {};
    beforeEach(()=>{
        props.location = {search:''}
    })
    it('snapshot test',()=>{
        let wrapper = shallow(<Layout {...props}/>)
        expect(wrapper).toMatchSnapshot()
    })
})