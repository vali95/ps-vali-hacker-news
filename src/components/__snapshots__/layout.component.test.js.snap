// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component test snapshot test 1`] = `
<div
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <table>
    <thead>
      <tr
        className="head"
      >
        <th
          key="Comments"
        >
          Comments
        </th>
        <th
          key="Vote Count"
        >
          Vote Count
        </th>
        <th
          key="UpVote"
        >
          UpVote
        </th>
        <th
          key="News Details"
        >
          News Details
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
  <div
    className="nav"
  >
     | 
  </div>
  <hr />
  <Graph
    data={Array []}
  />
</div>
`;
